SimTracker/TrackerHitAssociation/plugins/ClusterSLOnGPU.cu

```diff
@@ -9,23 +9,21 @@
 
 #include "ClusterSLOnGPU.h"
 
-using ClusterSLGPU = trackerHitAssociationHeterogeneousProduct::ClusterSLGPU;
-using Clus2TP = ClusterSLGPU::Clus2TP;
+using ClusterSLView = trackerHitAssociationHeterogeneous::ClusterSLView;
+using Clus2TP = ClusterSLView::Clus2TP;
 
 // #define DUMP_TK2
 
 __global__ void simLink(const SiPixelDigisCUDA::DeviceConstView* dd,
                         uint32_t ndigis,
-                        clusterSLOnGPU::HitsOnGPU const* hhp,
-                        ClusterSLGPU const* slp,
+                        TrackingRecHit2DSOAView const* hhp,
+                        ClusterSLView sl,
```

Comment:
I suppose passing `ClusterSLView` by value here (and elsewhere) is intentional.

Reply:
Indeed.
It is a different modus operandi w/r/t the one used for Digis and Hits